src/OpenCL/Extensions/Silk.NET.OpenCL.Extensions.INTEL/Enums/AdvancedMotionEstimationClMotionDetectDesc.gen.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System;
+using Silk.NET.Core.Attributes;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.OpenCL.Extensions.INTEL
+{
+    [NativeName("Name", "cl_intel_advanced_motion_estimation.cl_motion_detect_desc_intel")]
```

Comment:
It looks like the type name is actually cl_motion_detect_desc_intel and this is just sort of "namespaced" with the extension name. did we want to account for this?

Reply:
Well, it's one of the parts where the spec is inconsistent, especially so extensions coming from Intel, those are the most "special". Basically, this is used only by Intel - can be used but only in a form of special "if vendor == Intel" handling.